x-pack/test/api_integration/apis/endpoint/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function endpointAPIIntegrationTests({ loadTestFile }: FtrProviderContext) {
+  describe('Endpoint', function() {
+    this.tags('ciGroup7');
+    loadTestFile(require.resolve('./resolver'));
```

Comment:
@oatkiller - my mistake - you don't need the CI tag in line 11.

Reply:
gotcha. will remove